faqs/atm7_http.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to enable HTTP on >1.12
+search: atm7 http
+---
+
+**Singleplayer**
+1. Refer to your launcher, you are looking for the game folder.
+2. Open the folder titled `saves`.
+3. Find your world folder, and open it.
+4. Find a folder titled `serverconfig`, and open it.
+5. Exit the game (or run `stop` on the server).
+6. Search for a file named `computercraft-server.toml`, and open it in a text editor.
+7. Search the file for a section starting with `[http]`.
+8. Find `enabled` within this section, and set it to `true`.
+9. Save the file and close the editor.
+
+**Multiplayer**
+If you're a player on the server, you will need to ask the server owner/operator to make these changes for you. 
+
+(If you are a server owner)
+1. Stop the server (run the command `stop`).
+2. Go into the files of your server, then go into ``world``
+3. Open ``serverconfig``
+4. Edit ``computercraft-server.toml``
+5. Search for ``http``
+6. Change ``enabled`` or ``enable`` to true
```

Comment:
```suggestion
5. Search the file for a section starting with `[http]`.
6. Find `enabled` within this section, and set it to `true`.
7. Save the file and close the editor.
```

It seems a bit strange to me that this doesn't match the previous steps - this bit should be the same!